README.md

```diff
@@ -8,11 +8,12 @@ With the repository containing many different packages related to iModel.js, eac
 
 List of packages within this repository:
 
-| Name                         | Folder                 |
-| ---------------------------- | ---------------------- |
-| @bentley/imodel-select-react | packages/imodel-select |
-| @bentley/tree-widget-react   | packages/tree-widget   |
-|                              |                        |
+| Name                             | Folder                     |
+| -------------------------------- | -------------------------- |
+| @bentley/imodel-select-react     | packages/imodel-select     |
```

Comment:
@calebmshafer should we start linking to the readmes of the pkgs like so:
```
| Name                         | Folder                                                     |
| ---------------------------- | ---------------------------------------------------------- |
| @bentley/imodel-select-react | [packages/imodel-select](packages/imodel-select/README.md) |

Reply:
Yea, that sounds like a good suggestion to me.  Save people a few clicks

Reply:
its not technically the folder, but leads to where they can find out more about the pkg. we could also add npm badges right next to it. im thinking like how rush does it with their readme https://github.com/microsoft/rushstack/blob/master/README.md

Reply:
Updated please!